src/Microsoft.Dnx.Tooling/Building/BuildManager.cs

```diff
@@ -235,6 +236,11 @@ private bool BuildInternal(string projectPath)
 
                         var root = _currentProject.ProjectDirectory;
 
+                        if(_currentProject.PackageFiles != null && _currentProject.PackageFiles.Any())
```

Comment:
I wonder if the format-on-save extension has been updated for VS2015 now .... ;)